uniprot-search/src/main/java/org/uniprot/store/search/domain2/SearchFieldsLoader.java

```diff
@@ -21,9 +21,9 @@
 public class SearchFieldsLoader implements SearchItems, SearchFields {
     private List<SearchItem> searchItems = new ArrayList<>();
```

Comment:
Are we using this search items? Because this class is "SearchFields"Loader, I did not expect SearchItems in it. If we need it, Maybe we could rename the class..